### PR TITLE
Update deps. Add sync to PG script. Add migration. Patch councils to …

### PR DESCRIPTION
**Pull request by awhite234**
…come from PG.

### CODE REVIEW

File: src/api/tasks/Tasks.controller.js [hunk 1]
```diff
@@ -549,7 +549,11 @@ const getRegistration = async (client, fsaId) => {
 
 const findCouncilByIdInArray = (id, allCouncils = []) => {
   logEmitter.emit("functionCall", "Tasks.controller", "findCouncilByIdInArray");
-  let out = allCouncils.find((council) => council._id === id);
+  let out = allCouncils.find((council) => {
+    console.log(council);
```
Comment (jj-perry) [inline src/api/tasks/Tasks.controller.js:553]: Should this line be removed now?

File: scripts/sync-council-table.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  let localCouncils = null;
+  logEmitter.emit("functionCall", "sync-council-table", "getLocalCouncils");
+
+  try {
+    await establishConnectionToMongo();
+
+    localCouncils = await lcConfigCollection
+      .find({
+        $and: [
+          { local_council_url: { $ne: "" } },
```
Comment (jj-perry) [inline scripts/sync-council-table.js:31]: Am I right in thinking this section only grabs the local councils (those with a url field) ? Do we not want to move the county councils across as well?

**Comment by paul-burnett** [comment]
Updated migrations to use registrations schema, as is used in azure environments.
Moved councils table to new councils schema

File: scripts/sync-council-table.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  for (const record of data) {
+    model = await getCouncilByUrl(record.local_council_url);
+
+    logEmitter.emit(
+        INFO,
+        `Done ${record.local_council_url}`
+    );
+
+    if (model === null) {
+      model = await createCouncil(data, transaction);
```
[the remark this answers is not in the document]
Comment (paul-burnett) [reply scripts/sync-council-table.js]: Yes, thanks fixed this

File: src/api/tasks/Tasks.controller.js (hunk 1 above)
Comment (paul-burnett) [reply src/api/tasks/Tasks.controller.js]: removed unnecessary line

File: scripts/sync-council-table.js (hunk 2 above)
Comment (paul-burnett) [reply scripts/sync-council-table.js]: Correct, removed this filter.  Thanks

**Comment by simon-says-test** [comment]
This PR is no longer required since data is moving instead to Cosmos

**Pull request closed without merging**